app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -140,12 +141,18 @@ public Result createSynapse(String synapseUserId) throws Exception {
         return createdResult(new SynapseProjectIdTeamIdHolder(study.getSynapseProjectId(), study.getSynapseDataAccessTeamId()));
     }
 
-    public Result deleteStudy(String identifier) throws Exception {
+    // since only admin can delete study, no need to check if return results should contain deactivated ones
+    public Result deleteStudy(String identifier, String physical) throws Exception {
         getAuthenticatedSession(ADMIN);
         if (studyWhitelist.contains(identifier)) {
             return forbidden(Json.toJson(identifier + " is protected by whitelist."));
         }
-        studyService.deleteStudy(identifier);
+        if ("true".equals(physical)) {
+            studyService.deleteStudy(identifier, true);
+        } else {
+            studyService.deleteStudy(identifier, false);
+        }
```

Comment:
studyService.deleteStudy(identifier, Boolean.valueOf(physical)); // would be terser